korman/properties/modifiers/render.py

```diff
@@ -500,3 +500,35 @@ def export(self, exporter, bo, so):
             if not region.control_region:
                 raise ExportError("{}: Not all Visibility Controls are set up properly in Visibility Set".format(bo.name))
             addRegion(exporter.mgr.find_create_key(plVisRegion, bl=region.control_region))
+
+
+fog_types = {
+    "none":     plFogEnvironment.kNoFog,
+    "linear":   plFogEnvironment.kLinearFog,
+    "exp2":     plFogEnvironment.kExp2Fog
+}
+
+class PlasmaFogEnvMod(PlasmaModifierProperties):
+    pl_id = "fogenv"
+
+    bl_category = "Render"
+    bl_label = "Fog Environment"
+    bl_description = "Adjust per-object fog settings"
+    bl_icon = "MAT_SPHERE_SKY"
+
+    fog_env = PointerProperty(name="Environment",
```

Comment:
This seems like a misnomer when you are referencing a world. It's also confusing when accessing `bo.plasma_modifiers.fogenv.fog_env` because of the redundant naming and inconsistent spelling.